rpm/podman.spec

```diff
@@ -187,7 +191,11 @@ when `%{_bindir}/%{name}sh` is set as a login shell or set as os.Args[0].
 Summary: Metapackage for setting up %{name} machine
 Requires: %{name} = %{epoch}:%{version}-%{release}
 Requires: gvisor-tap-vsock
+%if %{defined qemu}
 Requires: qemu
```

Comment:
Note that "qemu" is a really awkward dependency - it pulls in tons of emulators, spice-server, GTK, vte291 etc. -- all stuff that you really don't want on servers. `qemu-system-x86-core` or `qemu-kvm-core` exist everywhere and aren't so heavyweight.

Reply:
@Luap99 @baude ^

Reply:
We do not use emulation, AFAICT we call qemu-system-x86_64 on x86_64 and qemu-system-aarch64 on aarch64.
Then we also need `qemu-img` to resize the disk so whatever proves what we need should be fine.
```
$ rpm -q --whatprovides /usr/bin/qemu-img 
qemu-img-8.2.8-2.fc40.x86_64
$ rpm -q --whatprovides /usr/bin/qemu-system-x86_64
qemu-system-x86-core-8.2.8-2.fc40.x86_64
$ rpm -q --whatprovides /usr/bin/qemu-system-aarch64 
qemu-system-aarch64-core-8.2.8-2.fc40.x86_64

```